jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/src/test/java/org/jbpm/console/ng/wi/client/editors/deployment/descriptor/DeploymentDescriptorViewTest.java

```diff
@@ -207,4 +216,25 @@ public void changeDefaultLimitSerializationClassesValueToTrueIn7x() throws Excep
                 ( jarVersion < 7.0d && ! limitSerializationClasses )
                 || (jarVersion >= 7.0d && limitSerializationClasses ) );
     }
+
+    @Test
+    public void runtimeStrategiesSetting() throws Exception {
+        DeploymentDescriptorViewImpl deploymentDescriptorView = new DeploymentDescriptorViewImpl();
+        deploymentDescriptorView.setPersistenceModeDropdown(persistenceModeDropdown);
```

Comment:
these setter methods aren't needed, you can reuse the `listBox` variable to verify. The only thing is that you would need to break this test into three separated ones, one for each dropdown.